src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -315,7 +315,7 @@ def load_arguments(self, _):
         c.argument('pod_cidrs')
         c.argument('service_cidrs')
         c.argument('load_balancer_managed_outbound_ipv6_count', type=int)
-        c.argument('enable_pod_security_policy', action='store_true')
+        c.argument('enable_pod_security_policy', action='store_true', is_preview=True, deprecate_info=c.deprecate(hide=True))
```

Comment:
`is_preview=True` is not necessary for deprecated parameters
```suggestion
        c.argument('enable_pod_security_policy', action='store_true', deprecate_info=c.deprecate(hide=True))
```